Add survey data

aays opened the pull request:
Not sure how much we'll be able to use this after all, but here it is all the same. I created two files: one with all survey data, and another with just quantitative answers. Addresses issue #12. 

I can also upload the script I used to make these as a separate file if needed - it's not super fancy:

```R
library(dplyr)
library(readr)

d1 <- read_csv('Desktop/Course-feedback-overall.csv') %>% 
  mutate(year = 2017)
d2 <- read_csv('Desktop/EEB313 Overall Course Feedback.csv') %>% 
  mutate(year = 2018)

surveys <- bind_rows(d1, d2) %>% 
  select(year, everything()) # reorder columns

surveys_quantitative <- surveys %>% 
  select_if(function(x) is.numeric(x))

write_csv(surveys, 'course-surveys.csv')
write_csv(surveys_quantitative, 'course-surveys-quantitative.csv')
```

aays committed:
add survey data

lwjohnst86 commented:
We're going to put this on hold until we hear back from Ethics.

lwjohnst86 commented on data/course-surveys.csv:
```diff
@@ -0,0 +1,25 @@
+2018,2018/12/06 12:07:14 AM EST,4,I think it's fine,3,4,It's fine,3,"Have more instructions on the final report, especially where to submit it","The datasets were less reliable, it is more difficult to figure out what we're doing.",4,5,5,N/A,The instructors understand students well and are very helpful.,Statistical analysis skill (especially on ANOVA); R programming skill,N/A,Yes,5
+2018,2018/12/07 3:09:42 AM EST,5,"If GitHub is going to be used, it is better to be taught earlier so by the time the projects started, they will have enough knowledge of Git.",2,5,GitHub assessment,4,not include GitHub,"The only difference is that we had now time to work in the project during class, something I really appreciate",4,5,5,"If GitHub is used, focus a little more on it, not just 1 le ...
+2018,2018/12/07 4:23:25 PM EST,5,"Challenges were great, helped make sure I was on track during the lecture.",3,5,Some instructors went a bit fast/taught more for an audience that was more familiar with programming/R in general and it would have been nice to slow down and do a bit more explaining.,5,no idea,"having ample time in class to work on the final project.
+
```
There's a problem here. Seems the quote didn't work.

lwjohnst86 commented on data/course-surveys.csv:
```diff
@@ -0,0 +1,25 @@
+year,Timestamp,How would you rate the course *lecture* material overall?,"How can the course material be improved (for example, the pacing, structure, content, concepts, and/or challenges)?",The ratio of instructors to students was:,How would you rate the instructors overall?,What can the instructors improve upon?,How would you rate the final project?,How could the final project be improved?,Whic ...
+2017,2017/12/05 3:00:44 PM EST,4,More challenges. Some of the GitHub stuff isn't on the online lectures which made it hard to remember what we learned.,3,4,Pacing,4,More in-depth outline of what is needed and more time to work on the project,"My group was easy to work with, the biggest difference is actually using R and GitHub",4,5,3,Make it a full year course and ask more ecological/biological q ...
```
Do you mind stripping the time component of the stamp? And just keep the date?

lwjohnst86 commented on data/course-surveys.csv:
```diff
@@ -0,0 +1,25 @@
+year,Timestamp,How would you rate the course *lecture* material overall?,"How can the course material be improved (for example, the pacing, structure, content, concepts, and/or challenges)?",The ratio of instructors to students was:,How would you rate the instructors overall?,What can the instructors improve upon?,How would you rate the final project?,How could the final project be improved?,Whic ...
```
Could you make the columns easier to read/use? (e.g. "Date", "OverallRating", "ImprovementsFeedback", etc)

lwjohnst86 commented on data/course-surveys-quantitative.csv:
```diff
@@ -0,0 +1,23 @@
+year,How would you rate the course *lecture* material overall?,The ratio of instructors to students was:,How would you rate the instructors overall?,How would you rate the final project?,Did the lectures prepare you adequately for the final project?,How relevant do you think the concepts discussed during class will be for your future work/studies?,Did you need to spend more or less time and effor ...
```
Could you use better column headers?

lwjohnst86 commented:
Thanks @aays! Could you make those changes I requested, as well as include the R code as a file in the PR? Thanks!!!! :D 

Oh, and by the way, Ethics seems to be fine with using the information for research :)

aays committed:
update files and add script

aays committed:
add raw data files

aays commented:
@lwjohnst86 Thanks for the review! The files have been updated and I've added the script as well. I also added the raw files in a separate commit just in case.

aays committed:
rename 2018 raw file

aays committed:
clear out old filename

lwjohnst86 reviewed (approved):
Looks great!
